samples/sample_copy_path.jl

```diff
@@ -0,0 +1,73 @@
+
+using Cairo
+
+function common_header()
+	## header to provide surface and context
```

Comment:
indentation is done, how sublime2 does it. What is the convention on indentation?